utils/bitcoin.ts

```diff
@@ -7,10 +7,10 @@ import { CONST } from '../src/consts';
 const BigNumber = require('bignumber.js');
 const reverse = require('buffer-reverse');
 
-export const btcToSatoshi = (btc: number, precision: number | null = null) => {
+export const btcToSatoshi = (btc: number, precision: number | null = null): number => {
   const satoshis = new BigNumber(btc).multipliedBy(CONST.satoshiInBtc);
   if (precision === null) {
-    return satoshis;
+    return satoshis.toNumber();
```

Comment:
It'd be enough to add `.toNumber()` in the 11th line and remove it from the 16th line.

Reply:
done